src/pybind/mgr/dashboard/frontend/src/app/ceph/new-dashboard/dashboard/dashboard.component.ts

```diff
@@ -1,8 +1,41 @@
-import { Component } from '@angular/core';
+import { Component, OnInit } from '@angular/core';
+
+import { ConfigurationService } from '~/app/shared/api/configuration.service';
+import { OrchestratorService } from '~/app/shared/api/orchestrator.service';
+import { SummaryService } from '~/app/shared/services/summary.service';
 
 @Component({
   selector: 'cd-dashboard',
   templateUrl: './dashboard.component.html',
   styleUrls: ['./dashboard.component.scss']
 })
-export class DashboardComponent {}
+export class DashboardComponent implements OnInit {
+  detailsCardData: any = {};
+  // function for keyvalue order
+  zero: any = () => 0;
```

Comment:
why is this?

Reply:
@nizamial09 it is to keep the original order of the inserted elements into the dictionary, since ` | keyvalues ` it reordes the elements alphabetically

Reply:
If there is a better way to do it, could you let me know? Thanks @nizamial09

Reply:
Hmm I think no. I didn't knew `keyvalue` had a default sorting. Btw instead of just zero, its better to call the function `returnZero`.

Reply:
got rid of the `*ngFor` and switched to an html description list